CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Release report: https://github.com/wazuh/wazuh/issues/15504
 
 ### Added
 
+- Add test to check the Syscollector configuration. ([#3584](https://github.com/wazuh/wazuh-qa/pull/3584)) \- (Framework + Tests)
```

Comment:
This will be released in 4.4.2, please move it to 4.4.2 section

Reply:
Done in f1426cd